lib/pool.js

```diff
@@ -252,28 +252,15 @@ class WorkerNodes extends EventEmitter {
      * @returns {Worker}
      */
     pickWorker() {
-        if (this.options.lazyStart) {
```

Comment:
As far as I can tell, `lazyStart` only matters for startup,
i.e. do we start up to `minWorkers` or `maxWorkers`, so IMO this block can be simplified.

Also previous code would break if the worker took time to startup, that is because the called of `pickWorker()` in the `lazyStart` option would receive a worker that was not yet `isProcessAlive` (line 259 & 263) and would try  to dispatch work to it in `processQueue`:
```es6
            const worker = this.pickWorker();
            if (worker) {
                this.dispatchTaskTo(worker);
            }
```

Reply:
Ok, that wasn't entirely true, but it still can be simplified to:
```es6
        if (this.canStartWorker() && (!this.options.lazyStart || !worker)) {
            this.startWorker();
        }
```